code/modules/power/apc.dm

```diff
@@ -84,12 +84,13 @@
 	var/has_electronics = 0 // 0 - none, 1 - plugged in, 2 - secured by screwdriver
 	var/overload = 1 //used for the Blackout malf module
 	var/beenhit = 0 // used for counting how many times it has been hit, used for Aliens at the moment
-	var/mob/living/silicon/ai/occupant = null
+	var/mob/living/silicon/ai/occupier = null

```

Comment:
Is there a reason to change this variable name? Occupant is the unofficial standard for things that hold a mob in them.